osbs/utils/yaml.py

```diff
@@ -62,7 +63,13 @@ def read_yaml(yaml_data, schema, package=None):
     except jsonschema.SchemaError:
         logger.error('invalid schema, cannot validate')
         raise
-    except jsonschema.ValidationError:
+    except jsonschema.ValidationError as exc:
+        logger.debug("schema validation error: %s",
+                     exc.message)  # pylint: disable=exception-message-attribute
```

Comment:
just `exc`

Reply:
There is exc.message here because according to the documentation, message attribute has a more readable message explaining the error than str(exc)

Reply:
Here is the documentation
https://python-jsonschema.readthedocs.io/en/stable/errors/

Reply:
I would like to have full output in debug